CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+### 0.2.1 (2023/11/06)
```

Comment:
```suggestion
### 0.3.0 (2023/11/06)
```
i was hoping that patch versions could be used by downstream to add customizations.